lib/enums/query/query_presets.py

```diff
@@ -81,9 +83,7 @@ class QueryPresetsString(QueryPresets):
     Enum class which holds string comparison operators
     """
 
-    ANY_IN = auto()
     MATCHES_REGEX = auto()
-    NOT_ANY_IN = auto()
```

Comment:
Why have we dropped this?
Whilst you can use regex, doing multiple groups with regex is painful such as `ANY_IN ("foo", "bar")` is easier to read / parse?